combat-trainer.lic

```diff
@@ -284,6 +276,21 @@ class LootProcess
     pause 0.2 until DRRoom.dead_npcs.empty?
   end
 
+  def consume(mob_noun, game_state, wounds)
+    check_thanatology(DRRoom.dead_npcs.first, 'consume', game_state)
+    # Flags.add('necro-spellcast', '^You feel fully prepared to cast your spell\.')
+    if game_state.casting
+      bput('release spell', '')
+      game_state.casting = false
+    end
+    bput("prepare cf", @settings.prep_messages)
+    game_state.casting = true
+    waitcastrt?
```

Comment:
In that case, you may want to add a check here to NOT release if the current spell is consume flesh.